app/controllers/content_items_controller.rb

```diff
@@ -26,8 +26,6 @@ def present(content_item)
     presenter_name = content_item['schema_name'].classify + 'Presenter'
     presenter_class = Object.const_get(presenter_name)
     presenter_class.new(content_item)
-  rescue NameError
```

Comment:
Looking at Errbit now, I can see this error is being raised on production:
eg `No support for schema "special_route"`

Worth some separate investigation.